src/common/low_precision_transformations/include/low_precision/rt_info/precisions_attribute.hpp

```diff
@@ -31,8 +31,7 @@ using PrecisionsAttributePtr = std::shared_ptr<PrecisionsAttribute>;
 
 class LP_TRANSFORMATIONS_API PrecisionsAttribute : public SharedValueAttribute<PrecisionsSharedValue> {
 public:
-    static std::vector<ngraph::element::Type> defaultPrecisions;
-    PrecisionsAttribute(const std::vector<ngraph::element::Type>& precisions = defaultPrecisions);
+    PrecisionsAttribute(const std::vector<ngraph::element::Type>& precisions = ngraph::pass::low_precision::LayerTransformation::getDefaultPrecisions());
```

Comment:
remove default argument and dependency on `layer_transformation.hpp` (`#include "low_precision/layer_transformation.hpp"`), please